app/assets/javascripts/auth/registrations.js

```diff
@@ -0,0 +1,57 @@
+/* global layout_resizer */
+
+$(document).ready(function () {
+  $('#add_application_token_btn').on('click', function (_event) {
+    $('#add_application_token_form').toggle(400, 'swing', function () {
+      if ($('#add_application_token_form').is(':visible')) {
+        $('#add_application_token_btn i').addClass('fa-minus-circle');
+        $('#add_application_token_btn i').removeClass('fa-plus-circle');
+        $('#application_token_application').val('');
+        $('#application_token_application').focus();
+      } else {
+        $('#add_application_token_btn i').removeClass('fa-minus-circle');
+        $('#add_application_token_btn i').addClass('fa-plus-circle');
+      }
+      layout_resizer();
+    });
+  });
+});
+
+jQuery(function () {
```

Comment:
Should be changed to, for not relying on the global alias for jQuery:
jQuery(function ( $ ) {

reference: http://api.jquery.com/jquery/#jQuery3